readme.md

```diff
@@ -586,6 +586,7 @@
 - [Ad-Free](https://github.com/johnjago/awesome-ad-free#readme) - Ad-free alternatives.
 - [Esolangs](https://github.com/angrykoala/awesome-esolangs#readme) - Programming languages designed for experimentation or as jokes rather than actual use.
 - [Prometheus](https://github.com/roaldnefs/awesome-prometheus#readme) - Open-source monitoring system.
+- [Prometheus-Alerts](https://github.com/samber/awesome-prometheus-alerts) - Prometheus queries and rules for alerting.
```

Comment:
This should be indented with a tab and renamed to just `Alerts` here. See other examples of sub-topics.